src/ui/Facet/FacetUtils.ts

```diff
@@ -15,7 +15,7 @@ export class FacetUtils {
   }
 
   static getDisplayValueFromValueCaption(value: string, field: string, valueCaption: Record<string, string>) {
-    const returnValue = this.tryToGetTranslatedCaption(field, value);
+    const returnValue = this.tryToGetTranslatedCaption(field, value, false);
     return valueCaption[value] || returnValue;
```

Comment:
What if we adjust line 19 to just read from the `valueCaption` object, and return the original value if no caption is found? Wouldn't this be more aligned with the name of the method vs. using the localization?

```
return valueCaption[value] || value;
```

If we did this, what would be the impact of the behaviour change i.e., which components are calling `getDisplayValueFromValueCaption`?

Reply:
It would more effectively prevent the issue, but it would also be even more breaking. `@filetype`, `@objecttype` and `@month` fields would lose their localization. However, we could copy their localization to the `valueCaption` record.

What do you think?

Reply:
Yes, it could be a good approach. Wouldn't this already be the case for facet search to work on a dynamic facet configured on `filetype`, `objecttype` and `month` though?

If you type into the [filetype facet search](https://static.cloud.coveo.com/searchui/v2.10090/index.html#t=All&sort=relevancy), the requests contain a `caption` object. Are those values in the `valueCaption` object passed to the function?

Which makes me wonder ... how would we differentiate between our filetype values, and a customer indexing different values into the filetype field, which they might want to caption differently? Are value caption objects merged? or is the custom object used instead if specified. More of an aside, but maybe something to look into ...

Reply:
> we could copy their localization to the valueCaption record

Yes +1
I think it's what Atomic does as well and it differentiates between any localized string and localized value caption strings

Reply:
> Are value caption objects merged? or is the custom object used instead if specified. More of an aside, but maybe something to look into ...

They are merged in `FacetSearchController` (`addTypesCaptionsIfNecessary()` and `get captions()`).